roles/ansible_service_broker/vars/openshift-enterprise.yml

```diff
@@ -1,7 +1,7 @@
 ---
 __ansible_service_broker_registry_type: rhcc
 __ansible_service_broker_registry_name: rh
-__ansible_service_broker_registry_url: "https://registry.access.redhat.com"
+__ansible_service_broker_registry_url: "https://registry.redhat.io"
```

Comment:
@jwmatthews if this is what i think it is, i'm guessing it's going to be broken by this change (registry.redhat.io requires auth) so we're going to need to make it clear that the registry_user and registry_password fields are not optional.